packages/ui/src/components/AccountDetails.tsx

```diff
@@ -72,6 +72,29 @@ export const AccountDetails: FC<Props> = ({ defaultName, headerText, onBack, onC
         iconAsset={SvgAccountCardDetailsOutline}>
       </Header>
       <Box mx='s'>
+        <Box borderColor='gray.4'
+          borderRadius={3}
+          borderStyle='solid'
+          borderWidth={2}
+          mt='s'
+          p='s'>
+          <Flex mb='xs'>
+            <Icon Asset={SvgAlertCircle}
+              color='warning'
+              height={20}
+              width={20} />
+            <Box ml='s'>
+              <Text color='warning'
+                variant='b2m'>
+                Attention
+              </Text>
+            </Box>
+          </Flex>
+          <Text color='gray.1'
+            variant='b2m'>
+            Use your current wallet password to be able to add this account.
```

Comment:
Please change copy to: `Enter your current wallet password in order to add this account.`